enhancements/cluster-scope-secret-volumes/csi-driver-host-injections.md

```diff
@@ -118,53 +101,24 @@ the credentials, I want to stop updating content within the existing pods consum
-an OLM operator to be present.
-- Using OLM avoids further increasing the install payload size.
-- Using OLM avoids unexpected hiccups in the feature's installation impacting initial cluster install or cluster upgrade.
-
-#### CSI starting points
-
-**WE ARE GOING WITH EPHEMERAL MODE**
+For the developer preview release, the CSI driver will be installable via YAML manifests available on GitHub.
+This is meant to be a preview only option and allow the CSI driver to be installed on OpenShift outside of the normal release cycle.
 
-**NO SecretProvideClass IS NEEDED**
+When the CSI driver is ready for tech preview, it will be installed on OpenShift via the cluster storage operator and an associated delivery operator - see the [CSI driver install proposal](/enhancements/storage/csi-driver-install.md).
 
-With those simplifying decisions, a CSI driver with those characteristics still meets our needs.  And the storage
-SMEs recommend that approach to us.
 
-Crafting the CSI driver in this way also helps us deal with SELinux.  Specifically (thanks to @jsafrane for details):
+#### CSI starting points
 
-- when CRI-O starts any container, it updates the SELinux labels on all volumes for that container, where those labels are unique to the container
-- in the context of k8s, multiple containers within a pod have the same filesystem access
-- so, only that specific pod can read files on that volume
-- however, a file can only have one SELinux label
-- so two or more pods cannot access the same file
-- but if each pod has its own file, its own copy, of the shared data, things work easily from a SELinux perspective
-- this is where using ephemeral CSI storage comes in
-- using ephemeral based storage allows for use of a tmpfs (virtual memory filesystem) for storage
-- tmpfs is fast and has negligible overhead
-- it will have to reconstructed if the Pod moves
-- but how it eases dealing with SELinux protection is the bonus we are taking
-- so each pod gets its own tmpfs
-- the kubelet leverages tmpfs in a similar fashion already
+The projected resource CSI driver will provision ephemeral volumes and take advantage of upstream's ephemeral CSI storage mechanisms.
+Ephemeral storage allows the CSI driver to create tmpfs filessytems for each pod, where the Secret/ConfigMap content will be copied.
+The tmpfs filesystems will automatically have appropriate SELinux labeling to ensure content does not leak between pods - the driver does not need to provision or manage these labels.
+The process will be similar to what exists in core Kubernetes when Secrets and ConfigMaps are mounted into a pod.
+Acutal implementation can borrow a significant amount of logic from the [secrets-store-csi-driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver), which is used to mount secrets from external "sealed" secret providers like [Vault](https://learn.hashicorp.com/tutorials/vault/kubernetes-secret-store-driver).
```

Comment:
This is a bit of a leap, though certainly that implementation leverages tmpfs mounts.  

And I think your simplification has removed the key points that were the original motivation / intent of the content you've replaced.
- tmpfs mounts per csi pod volume gets us "SELinux compliance" for free
- tmpfs mounts are fast, cheap resource wise
- tmpfs mounts == k8s ephemeral based storage volumes

I'm fine if you want to distill my verbose list into something more concise, but I think some form of these 3 bullets is what is needed here